File: haushaltsbuch/src/main/java/org/github/haschi/haushaltsbuch/api/_BeginneHaushaltsbuchführung.java
package org.github.haschi.haushaltsbuch.api;

import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung;
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Anweisung;

@Anweisung
public interface _BeginneHaushaltsbuchführung
{
    Aggregatkennung id();

    Inventar inventar();
}

File: haushaltsbuch/src/main/kotlin/org/github/haschi/haushaltsbuch/api/BeginneHaushaltsbuchführung.kt
package org.github.haschi.haushaltsbuch.api

import org.axonframework.commandhandling.TargetAggregateIdentifier
import org.github.haschi.haushaltsbuch.core.Inventar
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung

data class BeginneHaushaltsbuchführung(
        @TargetAggregateIdentifier val id: Aggregatkennung,
        val inventar: Inventar)
